test/unit/repositories/test_usage_status.py

```diff
@@ -17,235 +22,421 @@
-    """
-    usage_status_id = str(ObjectId())
-    session = MagicMock()
-    test_helpers.mock_find_one(database_mock.usage_statuses, None)
-    retrieved_usage_status = usage_status_repository.get(usage_status_id, session=session)
-
-    assert retrieved_usage_status is None
-    database_mock.usage_statuses.find_one.assert_called_once_with(
-        {"_id": CustomObjectId(usage_status_id)}, session=session
-    )
-
-
-def test_delete(test_helpers, database_mock, usage_status_repository):
-    """Test trying to delete a usage status"""
-    usage_status_id = str(ObjectId())
-    session = MagicMock()
-
-    test_helpers.mock_delete_one(database_mock.usage_statuses, 1)
-
-    # Mock `find_one` to return no child catalogue item document
-    test_helpers.mock_find_one(database_mock.items, None)
-
-    usage_status_repository.delete(usage_status_id, session=session)
-
-    database_mock.usage_statuses.delete_one.assert_called_once_with(
-        {"_id": CustomObjectId(usage_status_id)}, session=session
-    )
-
-
-def test_delete_with_an_invalid_id(usage_status_repository):
-    """Test trying to delete a usage status with an invalid ID"""
-    usage_status_id = "invalid"
-    session = MagicMock()
 
-    with pytest.raises(InvalidObjectIdError) as exc:
-        usage_status_repository.delete(usage_status_id, session=session)
-    assert str(exc.value) == "Invalid ObjectId value 'invalid'"
+        RepositoryTestHelpers.mock_find(
+            self.usage_statuses_collection,
+            [usage_status_out.model_dump() for usage_status_out in self._expected_usage_status_out],
+        )
 
+    def call_list(self) -> None:
+        """Calls the `UsageStatusRepo` `list method` method."""
+        self._obtained_usage_status_out = self.usage_status_repository.list(session=self.mock_session)
 
-def test_delete_with_a_non_existent_id(test_helpers, database_mock, usage_status_repository):
-    """Test trying to delete a usage status with a non-existent ID"""
-    usage_status_id = str(ObjectId())
+    def check_list_success(self) -> None:
+        """Checks that a prior call to `call_list` worked as expected."""
+        self.usage_statuses_collection.find.assert_called_once_with(session=self.mock_session)
+        assert self._obtained_usage_status_out == self._expected_usage_status_out
 
-    test_helpers.mock_delete_one(database_mock.usage_statuses, 0)
-    # Mock `find_one` to return no child catalogue item document
-    test_helpers.mock_find_one(database_mock.items, None)
 
-    with pytest.raises(MissingRecordError) as exc:
-        usage_status_repository.delete(usage_status_id)
-    assert str(exc.value) == f"No usage status found with ID: {usage_status_id}"
-    database_mock.usage_statuses.delete_one.assert_called_once_with(
-        {"_id": CustomObjectId(usage_status_id)}, session=None
-    )
+class TestList(ListDSL):
+    """Tests for listing usage statuses."""
 
+    # pylint: disable=duplicate-code
```

Comment:
```suggestion
```
We have been putting
```
# Expect some duplicate code inside tests as the tests for the different entities can be very similar
# pylint: disable=duplicate-code
```
at the top of our test files instead. Could you also remove the corresponding enable below.